R/cf.R

```diff
@@ -355,6 +417,112 @@ gen.block.array <- function(n, R, l, endcorr=TRUE) {
   return(list(starts = st, lengths = lens))
 }
 
+#' Computes the samples for reweighted correlation function
+#'
+#' @param cf `cf` object.
+#' @param rw `rw` object.
+#' @param nsamples Integer (Number of stochastic samples)
+#' @param boot.R Integer
+#' @param boot.l Integer
+#' @param seed Integer
+#' @param sim string
+#' @param endcorr boolean
+#' @export
+bootstrap_rw.cf <- function(cf, rw, nsamples=1, boot.R=400, boot.l=2, seed=1234, sim="geom", endcorr=TRUE) {
+  stopifnot(inherits(cf, 'cf_orig'))
+  stopifnot(inherits(rw, 'rw_orig'))
+  stopifnot(inherits(rw, 'rw_meta'))
+  stopifnot(inherits(cf, 'cf_indexed'))
+
+
+  ##We should also check that the cf object and the rw object contains the same gauge configurations
+
+
+  stopifnot(rw$conf.index == cf$conf.index)
+ 
+  stopifnot( nrow(cf$cf) ==nsamples*length(rw$conf.index) )
```

Comment:
```cf``` has no concept of stochastic samples in its current form, data should come in here already averaged

This means that if you use data with multiple stochastic samples per config, the averaging should be done before conversion into a ```cf``` object.